setup_configure.py

```diff
@@ -58,6 +58,7 @@ class EnvironmentOptions(object):
     def __init__(self):
         self.hdf5 = os.environ.get('HDF5_DIR')
         self.hdf5_version = os.environ.get('HDF5_VERSION')
+        self.mpi = os.environ.get('HDF5_MPI') == "ON"
```

Comment:
thank you for the feedback.

yes, I explicitly wanted an way to disable it again. e.g. right now the `--mpi` flag can only be deactivated again by removing the `h5config.pkl` pickle file which is a bit odd.

> For simplicity I agree just one 'true' value is better than a mid-sized set. 

ok then we leave it like it is right now. "ON" is true, everything else is false.